README.md

```diff
@@ -24,29 +24,9 @@ yarn start
 ```
 
 ## Status
-Despite lots of unchecked the most hard work is done. I'll add bindings to the rest of the components once I need them. Feel free to help me out and submit a PR.
-
-- [x] addLocaleData
-- [x] intlShape
-- [x] injectIntl
-- [x] defineMessages
-- [x] IntlProvider
-- [x] FormattedDate
-- [ ] FormattedDate (children-as-function)
-- [x] FormattedTime
-- [ ] FormattedTime (children-as-function)
-- [ ] FormattedRelative
-- [ ] FormattedRelative (children-as-function)
-- [ ] FormattedNumber
-- [ ] FormattedNumber (children-as-function)
-- [ ] FormattedPlural
-- [ ] FormattedPlural (children-as-function)
-- [x] FormattedMessage
-- [ ] FormattedMessage (children-as-function)
-- [x] DefinedMessage
-- [ ] DefinedMessage (children-as-function)
-- [ ] FormattedHTMLMessage
-- [ ] FormattedHTMLMessage (children-as-function)
+bs-react-intl should cover all of the [react-intl] 3.0.0 API. If you find anything missing, please file an issue.
 
 ## Usage
 See [`examples`](./examples) folder.
+
+[react-intl]: https://github.com/formatjs/react-intl
```

Comment:
Or pasted it in the wrong place :)

Reply:
No, should be fine. It is defined at the bottom and referenced twice above.

Reply:
Ah gotcha gotcha